src/cli/options/options_test.go

```diff
@@ -60,3 +60,50 @@ func (suite *OptionsUnitSuite) TestAddExchangeCommands() {
 	err := cmd.Execute()
 	require.NoError(t, err, clues.ToCore(err))
 }
+
+func (suite *OptionsUnitSuite) TestDisableConcurrencyLimiterFlag() {
+	tests := []struct {
+		name     string
+		args     []string
+		assertFn func(*testing.T, bool, string)
+		addFlag  bool
+	}{
+		{
+			name: "--disable-concurrency-limiter not set",
+			args: []string{"test"},
+			assertFn: func(t *testing.T, f bool, s string) {
+				assert.False(t, f, s)
+			},
+			addFlag: false,
+		},
+		{
+			name: "--disable-concurrency-limiter set",
+			args: []string{"test", "--disable-concurrency-limiter"},
```

Comment:
Flag names should always reference their const values.
```suggestion
			args: []string{"test", "--"+DisableConcurrencyLimiterFN},
```